Index: src/com/nothing/controller/ScoreController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/com/nothing/controller/ScoreController.java	(date 1577014791000)
+++ src/com/nothing/controller/ScoreController.java	(date 1577066269833)
@@ -89,7 +89,6 @@
     @RequestMapping("toClassScoByCid")
     public String toClassScoByCid( String testDH, String classId,StudentScore studentScore,HttpServletRequest request) throws ParseException{
         SimpleDateFormat formatter = new SimpleDateFormat("yyyy-MM-dd");
-
         Date testD = formatter.parse(testDH);
         studentScore.setScoreTime(testD);
         ClassVo classVo = (ClassVo) stuSer.findO(new ClassVo(), Integer.parseInt(classId));
@@ -104,7 +103,6 @@
         return "student/classScore";
     }
 
-
     @RequestMapping("toClassRepScoByCid")
     public String toClassRepScoByCid(String classId,String projectId,HttpServletRequest request){
         ClassVo classVo = (ClassVo) stuSer.findO(new ClassVo(), Integer.parseInt(classId));
@@ -116,13 +114,11 @@
 
     @ResponseBody
     @RequestMapping("toReplyMa")
-    public String toReply(String classId,String projectId,HttpServletRequest request){
-        ClassVo classVo = (ClassVo) stuSer.findO(new ClassVo(), Integer.parseInt(classId));
-        StudentProject project = (StudentProject)stuSer.findO(new StudentProject(), Integer.parseInt(projectId));
-       /* //List list = stuScoSer.isExistsTest(tId,cId,typeId,classId);
+    public String toReply(String classId,String projectId){
+       List list = stuScoSer.isExistsTestReply(classId,projectId);
         if (list.size()>0){
             return "no";
-        }*/
+        }
         return "yes";
     }
 
@@ -155,7 +151,6 @@
     @RequestMapping("toAddScore")
     @ResponseBody
     public String toAddScore(String tableList, StudentScore studentScore, String testDH, HttpSession session) throws ParseException{
-
         Emp e = (Emp)session.getAttribute("empId");
         SimpleDateFormat formatter = new SimpleDateFormat("yyyy-MM-dd");
         Date testD = formatter.parse(testDH);
@@ -177,7 +172,7 @@
             s.setTermId( studentScore.getTermId());
             s.setTestType(studentScore.getTestType());
             s.setScoreTime(testD);
-            //s.setEmpId(e.getEmpId());
+            s.setEmpId(e.getEmpId());
             System.out.println("循环内的考试对象"+s.toString());
             stuSer.addStu(s);
         }
@@ -201,7 +196,7 @@
             s.setStudId(Integer.parseInt(o.get("sid").toString()));
             s.setProjectId(Integer.parseInt(projectId));
             s.setReplyScoreRemark(o.get("rs").toString());
-            //s.setEmpId(e.getEmpId());
+            s.setEmpId(e.getEmpId());
             System.out.println("循环内的考试对象"+s.toString());
             stuSer.addStu(s);
         }
Index: src/com/nothing/controller/StudentControl.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/com/nothing/controller/StudentControl.java	(date 1577014791000)
+++ src/com/nothing/controller/StudentControl.java	(date 1577073081394)
@@ -102,6 +102,12 @@
         if(stuSelectFloor==null||"".equals(stuSelectFloor)){
             stuSelectFloor="";
         }
+        if(stuSelectPhone==null||"".equals(stuSelectPhone)){
+            stuSelectPhone ="";
+        }
+        if(stuSelectName==null||"".equals(stuSelectName)){
+            stuSelectName="";
+        }
         List<Map> list = stuSer.toStuConList(stuSelectName,stuSelectPhone,stuSelectCla,stuSelectFloor);
         int title = stuSer.conStu( stuSelectName ,stuSelectPhone, stuSelectCla, stuSelectFloor);
         JSONObject jsonObject = new JSONObject();
@@ -200,7 +206,7 @@
                  stuSer.updateStu(student);
              }
 
-            return "成功";
+            return "redirect: /stu/home";
     }
 
 
@@ -318,6 +324,8 @@
         return jsonArray;
     }
 
+
+
     //给学生分配班级
     @RequestMapping("toClassAddStu")
     @ResponseBody
@@ -327,8 +335,6 @@
         }catch (IndexOutOfBoundsException e){
 
         }
-
-
         return "";
     }
 
Index: src/com/nothing/controller/StuHolidayController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/com/nothing/controller/StuHolidayController.java	(date 1577014791000)
+++ src/com/nothing/controller/StuHolidayController.java	(date 1577068857918)
@@ -4,6 +4,7 @@
 import com.nothing.service.ActivitiService;
 import com.nothing.service.EmpService;
 import com.nothing.service.StuSer.ActStuSer;
+import com.nothing.service.StuSer.StuSer;
 import com.nothing.vo.Sdudent.Student;
 import com.nothing.vo.emp.Emp;
 import com.nothing.vo.emp.JobsVo;
@@ -22,8 +23,7 @@
 import javax.servlet.http.HttpSession;
 import java.text.ParseException;
 import java.text.SimpleDateFormat;
-import java.util.Date;
-import java.util.List;
+import java.util.*;
 
 @Controller
 @RequestMapping("actStu")
@@ -40,6 +40,8 @@
     private ActStuSer actStuSer;
     @Resource
     private HistoryService historyService;
+    @Resource
+    StuSer stuSer;
 
     @RequestMapping("toApply")
     public String toApplyPage(){
@@ -58,7 +60,7 @@
         return "actStu/stuActMyJob";
     }
 
-    @RequestMapping("toMyJob")
+        @RequestMapping("toMyJob")
     public String toMyJobPage(){
         return "actStu/stuActMyJob";
     }
@@ -77,16 +79,35 @@
         jsonObject.put("count",title);
         return jsonObject;
     }
+
     //我的请假单中查看备注
     @ResponseBody
     @RequestMapping("lookComment")
     public JSONObject lookComment(int jobId) throws ParseException{
-            JSONObject jsonObject = new JSONObject();
+        //通过jobId查询历史变量对象
+               JSONObject jsonObject = new JSONObject();
                 //通过jobID查询历史变量对象
                 HistoricVariableInstance hvi = historyService.createHistoricVariableInstanceQuery().variableValueEquals("jobId", jobId).singleResult();
                 //获取流程实例id （查询历史批注）
-                List<Comment> commentList = taskService.getProcessInstanceComments(hvi.getProcessInstanceId());
+                List<Comment>  commentList =new ArrayList<>();
+                    System.out.println("hvi:"+hvi);
+                    try{ commentList =taskService.getProcessInstanceComments(hvi.getProcessInstanceId());
+                    }catch (NullPointerException e){
+
+                    }
+                    List list=new ArrayList();
+                    for(Comment comment:commentList){
+                        Map dataMap=new HashMap();
+                        Emp emp   = (Emp) stuSer.findO(new Emp(),Integer.parseInt(comment.getUserId()));
+                        dataMap.put("empName",emp.getEmpName());
+                        dataMap.put("fullMessage",comment.getFullMessage());
+                        dataMap.put("sj",comment.getTime());
+                        list.add(dataMap);
+                        }
+                jsonObject.put("code",0);
+                jsonObject.put("msg","");
+                jsonObject.put("data",list);
+                jsonObject.put("count",list.size());
                 return jsonObject;
-
     }
 }
Index: src/com/nothing/service/impl/ActivitiServiceImpl.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/com/nothing/service/impl/ActivitiServiceImpl.java	(date 1577014791000)
+++ src/com/nothing/service/impl/ActivitiServiceImpl.java	(date 1577072743045)
@@ -31,7 +31,7 @@
 import java.util.zip.ZipOutputStream;
 
 @Service
-public class ActivitiServiceImpl extends BaseDao implements ActivitiService {
+public class ActivitiServiceImpl extends BaseDao implements ActivitiService{
     @Resource
     private RepositoryService repositoryService;
     @Resource
Index: src/com/nothing/service/StuSer/StuScoSerImp.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/com/nothing/service/StuSer/StuScoSerImp.java	(date 1577014791000)
+++ src/com/nothing/service/StuSer/StuScoSerImp.java	(date 1577064150617)
@@ -87,8 +87,8 @@
     }
 
     @Override
-    public List isExistsTestReply(String classId, String projectId) {
-        return null;
+    public List isExistsTestReply(String classId, String projectId){
+        return listBySQL("select * from stureplyscore where exists(select *,classId from stureplyscore left join student s using(studId) left join studentproject using(projectId) where classId = "+classId+" and projectId = "+projectId+")");
     }
 
 }
Index: web/WEB-INF/jsp/actStu/stuActMyJob.jsp
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- web/WEB-INF/jsp/actStu/stuActMyJob.jsp	(date 1577014791000)
+++ web/WEB-INF/jsp/actStu/stuActMyJob.jsp	(date 1577072743054)
@@ -26,28 +26,23 @@
 
 <div class="layui-row" style="height:45%">
     下面部分的
-    <div class="layui-tab" id="commentTab">
+    <div class="layui-tab">
         <ul class="layui-tab-title">
             <li>批注</li>
-        </ul>
-        <div class="layui-tab-content">
-            <div class="layui-tab-item layui-show">
-                <table id="comment" lay-filter="test"></table>
-            </div>
-        </div>
-    </div>
-    <div class="layui-tab" id="progressTab">
-        <ul class="layui-tab-title">
             <li>进度</li>
         </ul>
         <div class="layui-tab-content">
             <div class="layui-tab-item layui-show">
-                <table id=" progress" lay-filter="test"></table>
+                <table id="commentT" lay-filter="test"></table>
+            </div>
+            <div class="layui-tab-item">
+                <table id="progressT" lay-filter="test"></table>
             </div>
         </div>
     </div>
 </div>
 
+
 <script src="${pageContext.request.contextPath}/layui/layui.js">
 
 </script>
@@ -113,27 +108,47 @@
             switch(obj.event){
                 case 'comment':
                     table.render({
-                        elem: '#demo'
-                        ,toolbar: '#toolbarDemo' //开启头部工具栏，并为其绑定左侧模板
+                        elem: '#commentT'
                         ,url: '${pageContext.request.contextPath}/actStu/lookComment?jobId='+jId //数据接口
                         ,title: '批注'
                         ,cols: [[
-                            ,{field:'jobId',  title: '单据编号'}
+                            ,{field:' ',  title: '单据编号',
+                                templet: function (){
+                                    return jId;
+                                }
+                            }
                             , {
                                 field: ' ', title: '单据名称',
                                 templet: function (){
                                     return name;
                                 }
                             }
-                            ,{field:'sj',  title: '审批时间 '}
-                            ,{field:'userId', title:   '审批人 '}
+                            ,{field:'sj',  title: '审批时间 ',
+                                templet: function (row){
+                                    return createTime(row.sj);
+                                }
+                            }
+                            ,{field:'empName', title:   '审批人 '}
                             ,{field:'fullMessage', title:'批注内容'}
                         ]]
                     });
                     break;
+                case 'progress':
+                    window.open('${pageContext.request.contextPath}/lookMyJob?jobId='+jId,'self');
+                    break;
                 }
         });
     })
+
+    layui.use(['element','tab'], function(){
+        var element = layui.element;
+        var tab = layui.tab;
+        tab.render({
+            elem: '#commodity',
+            toolbar: '#toolbarDemo5'
+        })
+    });
+
     function process(v){
         if(v==1){
             return "审批中"
@@ -144,6 +159,16 @@
         }
         return""
     }
+    function createTime(v){
+        var date = new Date(v);
+        var y = date.getFullYear();
+        var m = date.getMonth()+1;
+        m = m<10?'0'+m:m;
+        var d = date.getDate();
+        d = d<10?("0"+d):d;
+        var str = y+"-"+m+"-"+d;
+        return str;
+    }
 
 </script>
 
@@ -151,9 +176,9 @@
     <button type="button" class="layui-btn layui-btn-s" lay-event="comment">
        查看批注
     </button>
-    <button type="button" class="layui-btn layui-btn-s" lay-event="jindu">
+   <%-- <button type="button" class="layui-btn layui-btn-s" lay-event="progress">
         查看进度
-    </button>
+    </button>--%>
 </script>
 </body>
 </html>
Index: web/WEB-INF/jsp/student/replyScore.jsp
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- web/WEB-INF/jsp/student/replyScore.jsp	(date 1577014791000)
+++ web/WEB-INF/jsp/student/replyScore.jsp	(date 1577066269842)
@@ -154,16 +154,13 @@
                 dataType:'text',
                 success:function (da){
                     var d = da
-                    if(''+d=='no'){
+                    if(d=='no'){
                         alert("此班级已有此答辩成绩存在");
                         return;
                     }else{
                         var classId =$('#classId').val();
-                        var courseId =$('#courseId').val();
-                        var testType =$('#testType').val();
-                        var termId  = $('#termId').val();
-                        var testDH = $('#testDH').val();
-                        self.location= "${pageContext.request.contextPath}/sco/toClassScoByCid?classId="+classId+"&courseId="+courseId+"&testType="+testType+"&termId="+termId+"&testDH="+testDH;
+                        var projectId = $('#projectId').val();
+                        self.location= "${pageContext.request.contextPath}/sco/toClassRepScoByCid?classId="+classId+"&projectId="+projectId;
                     }
                 }
             });
Index: web/WEB-INF/jsp/student/stuAdd.jsp
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- web/WEB-INF/jsp/student/stuAdd.jsp	(date 1577014791000)
+++ web/WEB-INF/jsp/student/stuAdd.jsp	(date 1577064364241)
@@ -16,7 +16,7 @@
     <link rel="stylesheet" href="${pageContext.request.contextPath}/layui/laydate/theme/default/laydate.css">
 </head>
 <body>
-<form class="layui-form" action="${pageContext.request.contextPath}/stu/update/add" method="post">
+<form class="layui-form" action="${pageContext.request.contextPath}/stu/update/add" method="post" target="_self">
     <div style="width: 100%;height: 69%;margin-top: 1%">
         <div style="width:33%;height:100%;float: left">
         <input type="hidden" value="0">
Index: web/WEB-INF/jsp/student/stuAllotClass.jsp
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- web/WEB-INF/jsp/student/stuAllotClass.jsp	(date 1577014791000)
+++ web/WEB-INF/jsp/student/stuAllotClass.jsp	(date 1577073081384)
@@ -53,16 +53,10 @@
             </div>
         </div>
 
-        <div class="layui-form-item" style="display: inline-block">
-            <label class="layui-form-label" style="width:100px">班级类型</label>
-            <div class="layui-input-block">
-                <input id="stuSelectType" type="text" name="stuSelectType" required  lay-verify="phone" placeholder="" autocomplete="off" class="layui-input">
-            </div>
-        </div>
-
         <button class="layui-btn layui-btn-sm" lay-event="isSelect">条件筛选</button>
     </div>
 </script>
+
 </body>
 
 <script src="${pageContext.request.contextPath}/layui/layui.js">
@@ -165,7 +159,6 @@
                 case 'isSelect':
                     var stuSelectName = $('input[name="stuSelectName"]').val();
                     var stuSelectPhone = $('input[name="stuSelectPhone"]').val();
-                    var stuSelectFloor = $("#stuSelectFloor").val();
                     table.reload('studentData',{
                         page: {
                             curr: 1 //重新从第 1 页开始
@@ -175,11 +168,11 @@
                         ,where:{
                             stuSelectName:stuSelectName,
                             stuSelectPhone:stuSelectPhone,
-                            stuSelectCla:" ",
+                            stuSelectCla:"",
                             stuSelectFloor:""
                         }
                     });
-                    table.reload('test');
+                    table.reload('studentData');
                     break;
             };
         });
@@ -198,7 +191,7 @@
                             employeesId+=data[i].studId+",";
                         }
                         layer.confirm('确定从此班级移除学生？', {icon: 3, title: '提示信息'},function (index){
-                            $.post('${pageContext.request.contextPath}/stu/toClassAddStu',{
+                            $.post('${pageContext.request.contextPath}/stu/toDel/cla',{
                                 studIds:employeesId,
                                 cid :0
                             },function(data){
@@ -228,7 +221,7 @@
                             stuSelectFloor:""
                         }
                     });
-                    table.reload('studentData');
+                    table.reload('test');
                     break;
             };
         });
Index: web/WEB-INF/jsp/student/stuMain.jsp
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- web/WEB-INF/jsp/student/stuMain.jsp	(date 1577014791000)
+++ web/WEB-INF/jsp/student/stuMain.jsp	(date 1577068857931)
@@ -266,16 +266,7 @@
         var str = y+"-"+m+"-"+d;
         return str;
     }
-    function process(v){
-        if(v==1){
-            return "审批中"
-        }else if(v==2){
-            return "审批通过"
-        }else if(v==3) {
-            return "审批否决";
-        }
-        return""
-    }
+
     function scType(v){
         if(v==1){
             return "笔试"
Index: web/WEB-INF/jsp/student/stuUpdate.jsp
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- web/WEB-INF/jsp/student/stuUpdate.jsp	(date 1577014791000)
+++ web/WEB-INF/jsp/student/stuUpdate.jsp	(date 1577064364250)
@@ -17,7 +17,7 @@
 </head>
 <body>
 
-<form class="layui-form" action="${pageContext.request.contextPath}/stu/update/up" method="post" lay-filter="gg">
+<form class="layui-form" action="${pageContext.request.contextPath}/stu/update/up" method="post" lay-filter="gg" target="_self">
     <input id="stuId" type="hidden" name="studId" lay-verify="required" autocomplete="off" class="layui-input" value="${stu.studId}">
     <div style="width: 100%;height: 69%;margin-top: 1%">
         <div style="width:33%;height:100%;float: left">
